index.html

```diff
@@ -358,7 +354,7 @@ <h3 id="w3c-coordination">W3C Groups</h3>
               <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>
             </dt>
   
-            <dd>
+            <dd><span class="todo">remove this?</span>
```

Comment:
Agreed. I don't think we explicitly need to mention TAG since TAG review is required as part of Horizontal Review.

Reply:
removed this pair : 9457ef0